assets/src/edit-story/components/panels/sizePosition.js

```diff
@@ -138,11 +139,17 @@ function SizePositionPanel({ selectedElements, onSetProperties }) {
     setBackgroundElement({ elementId: backgroundId });
   };
 
+  useEffect(() => {
+    const handler = setTimeout(() => updateProperties(), 300);
+
+    return () => clearTimeout(handler);
+  }, [updateProperties]);
```

Comment:
As discussed with @dvoytenko , there's already a race condition happening when you update a property then select a different element (the property will be applied to the newly selected element instead of the target you intended to update). This will worsen it. Since you're looking into this, do you mind also looking into that issue? Essentially when we issue a property update we need to ensure that its respective target ID is attached to it.

Reply:
Hey Wassim, can you or @jauyong please map this bug in an issue? I prefer to tackle it in another PR. BTW, I was able to reproduce the problem.

Reply:
Hey sorry it took so long but I created #610 for this. Might need some help taking a look and filling out the details / making sure it makes sense. 
@obetomuniz @wassgha